docs/examples.rst

```diff
@@ -152,3 +152,91 @@ BigBrain is a very large image (6572 × 7404 × 5711 voxels) reconstructed from
        white_right_327680.gii \
        classif/
    link-mesh-fragments --no-colon-suffix mesh_labels.csv classif/
+
+
+.. _Waxholm Rat:
+
+In the ``examples/Waxholm`` directory of the source distribution, you will find
+``WHS_SD_rat_T2star_v1.nii.gz``, sourced from
```

Comment:
makes sense. 

I am not too familiar with how git lfs work. Should I start from a new branch, so that the offending large file was never in history? Or would squash merge do the same thing?

I will be happy to do either (I will work on the former, i.e. a new branch + PR with the offending large file removed, and all references to the large file point to the existing colin template)

Reply:
After digging a bit , it [appears](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#removing-a-file-that-was-added-in-an-earlier-commit) that as long as the large file is not in the commit history, it will be removed (?)

As such, I will create a new branch and raise PR from the new branch. Once done, I will close this PR, and remove this branch, and ping you from the new branch.

Reply:
@xgui3783 actually I don't mind if the file is present in some old commits, by the nature of git-lfs the actual contents of the file are not part of the history, only a hash is, so the file will not be downloaded when users check out a commit where the file is deleted. Please don't waste your time rewriting history!

Reply:
understood. 

I will continue to work on this branch then. 

I will ping you again when I have a chance to work on the edits.